src/main/java/catan/services/util/play/EndTurnUtil.java

```diff
@@ -1,12 +1,44 @@
 package catan.services.util.play;
 
 import catan.domain.model.game.GameBean;
+import catan.domain.model.game.types.GameStage;
 import org.springframework.stereotype.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
 
 @Component
 public class EndTurnUtil {
+    private Logger log = LoggerFactory.getLogger(EndTurnUtil.class);
+
+    public Integer endTurnImplInPreparationStage(GameBean game, List<List<String>> initialBuildingsSet) {
```

Comment:
fixed to updateNextMoveInPreparationStage and deleted all steps from method that shouldn't be here